### PR TITLE
docs: update Bar documentation

### PR DESCRIPTION
**Pull request by InnaAtanasova**
## Description
update Bar documentation

### CODE REVIEW

File: stories/bar/bar.stories.js [hunk 1]
```diff
@@ -6,8 +6,39 @@ import '../../dist/segmented-button.css';
 export default {
     title: 'Components/Bar',
     parameters: {
-        description: `The Bar component is a container that holds titles, buttons and input controls. Its content is distributed in three areas - left, middle and right.
-The Bar has 2 modes - Desktop (default) and Tablet/Mobile (cozy).`,
+        description: `The bar component is a container that holds titles, buttons and input controls. The contents are distributed into three areas: left, middle and right. This component’s primary function is to display page **headers** and **footers**. It is mainly used to construct a **Page**, and acts as a building block for other components like **Dialog**, **Popover** etc.
```
Comment (stefanoScalzo) [inline stories/bar/bar.stories.js:9]: Its contents

File: stories/bar/bar.stories.js [hunk 2]
```diff
@@ -6,8 +6,39 @@ import '../../dist/segmented-button.css';
...
+        <br><br>
+
+**The bar component has two modes:**
+
+- Default (desktop)
```
Comment (stefanoScalzo) [inline stories/bar/bar.stories.js:14]: Should be enumerated

File: stories/bar/bar.stories.js [hunk 3]
```diff
@@ -120,12 +152,12 @@ export const tabletAndMobile = () => `
 tabletAndMobile.parameters = {
     docs: {
         iframeHeight: 100,
-        storyDescription: 'For Tablet and Mobile (Cozy mode) apply the <code>fd-bar--cozy</code> class to the container element.'
+        storyDescription: 'The cozy bar contains a back button, icon buttons, segmented button and product switch button. It is responsive to tablet and mobile screen sizes. To apply cozy mode, add the <code class="docs-code">--cozy</code> modifier class to the container element.'
```
Comment (stefanoScalzo) [inline stories/bar/bar.stories.js:155]: Should explain how it is responsive

Comment (InnaAtanasova) [reply stories/bar/bar.stories.js:155]: I don't think so, the provided text is pretty clear

File: stories/bar/bar.stories.js (hunk 1 above)
Comment (jessmetivier) [reply stories/bar/bar.stories.js:9]: "Its" or "The" both work, so I can agree with this comment.

File: stories/bar/bar.stories.js (hunk 2 above)
Comment (jessmetivier) [reply stories/bar/bar.stories.js:14]: Like 
1. Default
2. Cozy?
What exactly do you mean @stefanoScalzo ?

**Comment by InnaAtanasova** [comment]
> The only thing is the component type headers are still the same as before, am I crazy or did I send you the file with those changes? :S
> 
> **They would be:**
> Default
> Cozy
> Header
> Subheader
> Header With Subheader
> Footer
> Floating Footer

Hey @jessmetivier Friday I stopped updating the file because the build was failing (some test changes), that's why you don't see the new headers that you sent me.  I just pushed the file that you last sent me. You can check once again if everything is ok.

**Pull request merged**